src/contracts/core/AllocatorManagerStorage.sol

```diff
@@ -0,0 +1,120 @@
+
+    struct AllocatorDetails {
+        address delegationApprover;
+        bool isAllocator; // if true, then the address is an allocator. Needed beacause selectionApprover can be 0.
+    }
+
+    mapping(address => AllocatorDetails) internal _allocatorDetails;
+
+    /// @notice Mapping: staker => whether the staker is a post SDA staker (i.e. they have no left over state unmigrated from the DelegationManager).
+    mapping(address => bool) public isPostSDAStaker;
+
+    /**
+     * @notice Mapping: staker => allocator whom the staker is currently delegated to.
+     * @dev Note that returning address(0) indicates that the staker is not actively delegated to any allocator.
+     */
+    mapping(address => address) internal _delegatedTo;
+
+    /// @notice Mapping: staker => number of signed messages (used in `delegateToBySignature`) from the staker that this contract has already checked.
+    mapping(address => uint256) public stakerNonce;
+
+    /**
+     * @notice Mapping: delegationApprover => 32-byte salt => whether or not the salt has already been used by the delegationApprover.
+     * @dev Salts are used in the `delegateTo` and `delegateToBySignature` functions. Note that these functions only process the delegationApprover's
+     * signature + the provided salt if the allocator being delegated to has specified a nonzero address as their `delegationApprover`.
+     */
+    mapping(address => mapping(bytes32 => bool)) public delegationApproverSaltIsSpent;
+
+    /**
+     * TODO: edit
+     * @notice Global minimum withdrawal delay for all strategy withdrawals.
+     * In a prior Goerli release, we only had a global min withdrawal delay across all strategies.
+     * In addition, we now also configure withdrawal delays on a per-strategy basis.
+     * To withdraw from a strategy, max(minWithdrawalDelayBlocks, strategyWithdrawalDelayBlocks[strategy]) number of blocks must have passed.
+     * See mapping strategyWithdrawalDelayBlocks below for per-strategy withdrawal delays.
+     */
+    uint256 public minWithdrawalDelayBlocks;
+
+    /// @notice Mapping: hash of withdrawal inputs, aka 'withdrawalRoot' => whether the withdrawal is pending
+    mapping(bytes32 => bool) public pendingWithdrawals;
+
+    /// @notice Mapping: staker => cumulative number of queued withdrawals they have ever initiated.
+    /// @dev This only increments (doesn't decrement), and is used to help ensure that otherwise identical withdrawals have unique hashes.
+    mapping(address => uint256) public cumulativeWithdrawalsQueued;
+
+
+    /**
+     * @notice Minimum delay enforced by this contract per Strategy for completing queued withdrawals. Measured in blocks, and adjustable by this contract's owner,
+     * up to a maximum of `MAX_WITHDRAWAL_DELAY_BLOCKS`. Minimum value is 0 (i.e. no delay enforced).
+     */
+    mapping(IStrategy => uint256) public strategyWithdrawalDelayBlocks;
```

Comment:
feels like a possible opportunity to switch this from blocks to seconds/time, as the rest of the protocol has moved in that direction already